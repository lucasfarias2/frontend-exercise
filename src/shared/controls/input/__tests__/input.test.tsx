import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Input from '../input';

afterEach(cleanup);

describe('<Input />', () => {
  it('renders correctly', () => {
    const { container } = render(
      <Input
        value="test"
        placeholder="test"
        onChange={() => {
          return;
        }}
      />
    );
    expect(container).toMatchSnapshot();
  });
});
