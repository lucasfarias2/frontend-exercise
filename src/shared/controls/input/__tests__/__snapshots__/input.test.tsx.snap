// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input /> renders correctly 1`] = `
<div>
  <div
    class="ui__input"
  >
    <i
      class="ui__input__icon"
    />
    <input
      class="ui__input__bar"
      placeholder="test"
      value="test"
    />
  </div>
</div>
`;
